server/notification-providers/hostedsms.js

```diff
@@ -0,0 +1,52 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { UP } = require("../../src/util");
+
+class HostedSMS extends NotificationProvider {
+
+    name = "hostedsms";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
```

Comment:
It seems that `msg` is never used.

When `monitorJSON` and `heartbeantJSON` are null, `msg` is used for other general messages such as testing or certificate alert.